features/list-elements.yml

```diff
@@ -1,11 +1,15 @@
-draft_date: 2024-06-05
-name: <li>
-description: The `<li>` element TODO.
+name: <ol>, <ul>, and <li>.
+description: The elements to represent an ordered or unordered list.
 spec:
   - https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element
   - https://html.spec.whatwg.org/multipage/grouping-content.html#the-ol-element
   - https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element
 group: html
```

Comment:
Can you change this to belong to html-elements instead? This new group contains all HTML elements (same for your other PRs, if you can).

Reply:
@captainbrosset I opened https://github.com/web-platform-dx/web-features/pull/1467 to clean up the ones like this that have already merged.

Reply:
done